win32/tests/threadpool_win32_ut/threadpool_win32_ut.c

```diff
@@ -962,8 +964,9 @@ TEST_FUNCTION(threadpool_timer_start_with_NULL_timer_handle_fails)
     THANDLE_ASSIGN(THREADPOOL)(&threadpool, NULL);
 }
 
-/* Tests_SRS_THREADPOOL_WIN32_42_005: [ threadpool_timer_start shall allocate a context for the timer being started and store work_function and work_function_context in it. ]*/
+/* Tests_SRS_THREADPOOL_WIN32_42_005: [ threadpool_timer_start shall allocate memory for THANDLE_MALLOC(TIMER_INSTANCE) and store work_function and work_function_context in it. ]*/
 /* Tests_SRS_THREADPOOL_WIN32_42_006: [ threadpool_timer_start shall call CreateThreadpoolTimer to schedule execution the callback while passing to it the on_timer_callback function and the newly created context. ]*/
+/* Tests_SRS_THREADPOOL_WIN32_07_002: [ threadpool_timer_start shall initialize the a lock by calling srw_lock_ll_init. ]*/

```

Comment:
>initialize the a  [](http://example.com/codeflow?start=69&length=17)

"the a lock" should be corrected. "initialize the lock guarding the timer state" is better

Reply:
done